src/transactions/TransactionFrame.h

```diff
@@ -11,6 +11,7 @@
 #include "util/types.h"
 
 #include <memory>
+#include <optional>
```

Comment:
you're using `std::optional` in TxFrameBase, so this include needs to be included from there

Reply:
Moved to `TransactionFrameBase`, but for some reason, I get a linker error because of this change, fixed by https://github.com/sisuresh/stellar-core/commit/fbb21a8fe6eeceb2a4f7db2e0cd77b040f34b22c. Will look into this.

Reply:
That issue fixed itself after a full rebuild. The commit referenced above has been removed.